modules/rosa-classic-cluster-terraform-file-creation.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-classic-creating-a-cluster-quickly-terraform.adoc
+//
+
+ifeval::["{context}" == "rosa-classic-creating-a-cluster-quickly-terraform"]
+:tf-defaults:
+endif::[]
+:_content-type: PROCEDURE
+
+[id="rosa-classic-cluster-terraform-file-creation_{context}"]
+= Creating your Terraform files locally
+
+After you set up your link:https://console.redhat.com/openshift/token[offline {cluster-manager-first} token], you need to create the Terraform files locally to build your cluster. You can create these files by using the following code templates.
+
+.Procedure
+
+. Create the `main.tf` file by running the following command:
++
+[source,terminal]
+----
+$ cat<<-EOF>main.tf
+#
+# Copyright (c) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.20.0"
+    }
+    rhcs = {
+      version = ">= 1.5.0"
+      source  = "terraform-redhat/rhcs"
+    }
+  }
+}
+
+# Export token using the RHCS_TOKEN environment variable
+provider "rhcs" {}
+
+provider "aws" {
+  region = var.aws_region
+  ignore_tags {
+    key_prefixes = ["kubernetes.io/"]
+  }
+	default_tags {
```

Comment:
Should be at the same level as ignore_tags (remove leading spaces)

Reply:
I think this is just a gitHub issue. In the doc preview, it's lined up correctly:

![image](https://github.com/openshift/openshift-docs/assets/16167833/21d9e7b6-fcec-40e8-ba0a-bc08eca6b8dc)

In my editor too:

![image](https://github.com/openshift/openshift-docs/assets/16167833/9bd61135-8e79-4438-af45-5b6ca5a17e57)